vendor/github.com/VladimirMarkelov/clui/ctrlutil.go

```diff
@@ -213,7 +213,7 @@ func getLinearControlList(parent Control, fn func(Control) bool) []Control {
 // that has tab-stop feature on. Used by library when processing TAB key
 func NextControl(parent Control, curr Control, next bool) Control {
 	fnTab := func(c Control) bool {
-		return c.TabStop() && c.Visible() && c.Enabled()
+		return c.TabStop() && c.Parent().Visible() && c.Visible() && c.Enabled()
```

Comment:
This patch shouldn't be merged. It's being [reviewed upstream](https://github.com/VladimirMarkelov/clui/pull/104).